templates/CRM/Twingle/Page/Profiles.tpl

```diff
@@ -25,7 +25,7 @@
       <thead>
       <tr>
         <th>{ts domain="de.systopia.twingle"}Profile name{/ts}</th>
-        <th>{ts domain="de.systopia.twingle"}Properties{/ts}</th>
+        <th>{ts domain="de.systopia.twingle"}Selectors{/ts}</th>
```

Comment:
This actually used to be *Properties* as not only the `selector` property could be shown here, but maybe it's ok if there's nothing else displayed.

Reply:
Yes, the column was meant to display a variety of properties. But as I said, the list view becomes quite cluttered when a profile has many project_ids. So maybe it's better to dedicate this column to project_ids and add additional columns if there are more properties to show in the future.

The branch for the Twingle Shop integration adds another column for example:
![grafik](https://github.com/systopia/de.systopia.twingle/assets/24728186/340a7014-75a1-413a-b31c-ff0f411fca59)